### DISCUSSION TITLE
Issue with blog posts

### DISCUSSION THREAD

**Discussion by alexandrumeterez** [alshedivat/al-folio#1221] [Q&A]
Hi everyone!

I've been using this theme for a while and it works great, I love taking notes with it. I wrote a new blog post this morning however and all of a sudden, the names aren't parsed correctly anymore. I've attached an image. Because of this, the blog posts are not rendered properly (see the other image). I would love a bit of help! Thank you!

![image](https://user-images.githubusercontent.com/24506150/221891569-df5f4175-02a1-4114-a52d-09cc37f522d9.png)
![image](https://user-images.githubusercontent.com/24506150/221891737-5e463b67-103c-4333-a4ab-dc3ae398c885.png)

**Comment by george-gca** [comment]
Is this still happening?
